src/common/types.ts

```diff
@@ -0,0 +1,18 @@
+import { EnrichedOrder } from '@cowprotocol/cow-sdk'
+
+/**
+ * https://github.com/rndlabs/composable-cow/blob/main/src/ComposableCoW.sol
+ * Information about ComposableCoW conditional orders
+ *
+ * id - this parameter is specified when it's a conditional (parent) order
+ * parentId - this parameter is specified when it's a discrete (child) order
+ */
+export type ComposableCowInfo = {
+  id?: string
+  parentId?: string
+}
+
+export type OrderWithComposableCowInfo = {
+  order: EnrichedOrder
+  composableCowInfo?: ComposableCowInfo
+}
```

Comment:
Shouldn't this be part of the order instead?
I think that makes the interface simpler to work with

```suggestion
export type OrderWithComposableCowInfo = Order & {
  composableCowInfo?: ComposableCowInfo
}
```

Reply:
https://github.com/cowprotocol/cowswap/pull/2622#discussion_r1226533978